pkg/cmd/scan.go

```diff
@@ -60,12 +60,16 @@ func NewScanCmd() *cobra.Command {
 				)
 			}
 
-			outputFlag, _ := cmd.Flags().GetString("output")
-			out, err := parseOutputFlag(outputFlag)
+			outputFlag, _ := cmd.Flags().GetStringSlice("output")
+			if len(outputFlag) == 0 {
```

Comment:
Can we remove this if we specify the default to `[]string{output.Example(output.ConsoleOutputType)}` instead of `[]string{}` in `fl.StringSliceP` ?

Reply:
I used this approach because user may not want to send output to console at all. Also the current behavior is to not send output to console when a different output is used. Do you think we should change that ?

Reply:
But if you specify a default slice with console output, and some additional `--output` flags from CLI, they are not overwriting the whole default slice ?

Reply:
yeah you would need two slice or to empty the slice before appending

Reply:
It seems that it work well with a default slice :

```diff
diff --git a/pkg/cmd/scan.go b/pkg/cmd/scan.go
index c73fc265..e5f5bfb3 100644
--- a/pkg/cmd/scan.go
+++ b/pkg/cmd/scan.go
@@ -62,9 +62,6 @@ func NewScanCmd() *cobra.Command {
 			}
 
 			outputFlag, _ := cmd.Flags().GetStringSlice("output")
-			if len(outputFlag) == 0 {
-				outputFlag = append(outputFlag, output.Example(output.ConsoleOutputType))
-			}
 
 			out, err := parseOutputFlags(outputFlag)
 			if err != nil {
@@ -124,7 +121,7 @@ func NewScanCmd() *cobra.Command {
 	fl.StringSliceP(
 		"output",
 		"o",
-		[]string{},
+		[]string{output.Example(output.ConsoleOutputType)},
 		"Output format, by default it will write to the console\n"+
 			"Accepted formats are: "+strings.Join(output.SupportedOutputsExample(), ",")+"\n",
 	)

```

Without any flag we got : 

![2021-07-30_17-29](https://user-images.githubusercontent.com/6154987/127676288-04e8182d-20b7-4519-8deb-f53790e9825e.png)

With a json flag :

![2021-07-30_17-29_1](https://user-images.githubusercontent.com/6154987/127676299-0a886943-fb16-43aa-900c-9b005d29baf9.png)

This is exactly what we want for me, we should not change the default behavior when no flag are set, we should keep the console flag. If you do not want any output for me it's up to you to `> /dev/null`

Reply:
Yeah you're right, just applied your patch